inc/template-tags.php

```diff
@@ -158,47 +158,30 @@ function primer_paging_nav() {
 /**
  * Display navigation to next/previous post when applicable.
  *
- * @global WP_Post $post
+ * @uses the_post_navigation
+ * @uses primer_post_nav_text
+ *
  * @since  1.0.0
  */
 function primer_post_nav() {
 
-	global $post;
-
-	$previous = is_attachment() ? get_post( $post->post_parent ) : get_adjacent_post( false, '', true );
-	$next     = get_adjacent_post( false, '', false );
-
-	if ( ! $next && ! $previous ) {
-
-		return;
-
-	}
-
-	?>
-	<nav class="navigation post-navigation">
-
-		<h1 class="screen-reader-text"><?php esc_html_e( 'Post navigation', 'primer' ); ?></h1>
-
-		<div class="nav-links">
-
-		<?php if ( is_rtl() ) : ?>
-
-			<div class="nav-next"><?php next_post_link( '%link &larr;' ); ?></div>
-
-			<div class="nav-previous"><?php previous_post_link( '&rarr; %link' ); ?></div>
-
-		<?php else : ?>
-
-			<div class="nav-previous"><?php previous_post_link( '&larr; %link' ); ?></div>
-
-			<div class="nav-next"><?php next_post_link( '%link &rarr;' ); ?></div>
-
-		<?php endif; ?>
-
-		</div><!-- .nav-links -->
-
-	</nav><!-- .navigation -->
-	<?php
+	/**
+	 * Filter the next/prev navigation text
+	 *
+	 * @since NEXT
+	 *
+	 * @var array
+	 */
+	$navigation_text = (array) apply_filters( 'primer_post_nav_text', [
```

Comment:
@fjarrett Yea, I had the same thought. I figured less code/less filters would be better in the long run, but I can see the benefit of individual filters.